apis/v1alpha1/httproute_types.go

```diff
@@ -100,9 +100,24 @@ type HTTPRouteRule struct {
 	// match conditions that should be ANDed together.
 	// +optional
 	Matches []HTTPRouteMatch `json:"matches" protobuf:"bytes,1,rep,name=matches"`
-	// Filter defines what filters are applied to the request.
+
+	// Filters define the filters are applied to requests that match
+	// this rule.
+	//
+	// The effects of ordering of multiple behaviors are currently undefined.
+	// This can change in future based on feedback during the alpha stage.
+	//
+	// Conformance-levels at this level are defined based on the type of filter:
+	// - ALL core filters MUST be supported by all implementations.
+	// - Implementers are encouraged to support extended filters.
+	// - Vendor-specific custom filters have no API guarantees across implementations.
```

Comment:
I think we've been using implementation over vendor, but I could be wrong here.
```suggestion
	// - Implementation-specific custom filters have no API guarantees across implementations.
```